README.markdown

```diff
@@ -143,127 +137,176 @@ The following parameters are available in the ntp module:
 
 ####`autoupdate`
 
-**Deprecated:** This parameter determined whether the ntp module should be
-automatically updated to the latest version available.  Replaced by `package_ensure`.
+**Deprecated; replaced by the `package_ensure` parameter**
+Tells Puppet whether to keep the ntp module updated to the latest version available.
+Valid options: 'true' or 'false'
+Default value: 'false'
```

Comment:
Correction! To maximize legibility while avoiding this happening again in future revisions, we've removed those line breaks and added periods instead. New PR incoming...